ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/DistributingExecutor.java

```diff
@@ -89,6 +95,14 @@ public DistributingExecutor(
         .apply(executionContext, securityContext.getServiceContext())
         .inject(statement);
 
+    if (injected.getStatement() instanceof InsertInto) {
+      throwIfInsertOnInternalTopic(
+          statement.getConfig(),
```

Comment:
This ignores property overrides associated with the request, right? I'm wondering about the situation where a user has a valid reason to `INSERT INTO` an topic marked as internal by default, issues the request and sees an error, and now wants to override the `SYSTEM_INTERNAL_TOPICS_CONFIG` config to enable their use case. If this code ignores the statement override (which I believe it does), then they can't achieve the override without setting the config at a server level and restarting the KSQL server, which feels unnecessarily heavy-weight.

Reply:
Agree with @big-andy-coates, we should not let users override the setting. If the KSQL admin decided to hide and make those topics read-only, then is up to the admin to change the setting.

Reply:
That's fine, but can we be more explicit about it and add the property to https://github.com/confluentinc/ksql/blob/master/ksql-common/src/main/java/io/confluent/ksql/config/ImmutableProperties.java? The current implementation of this PR is inconsistent in that it does not allow overrides for `INSERT INTO` but it does for `INSERT VALUES`.